pkgs/by-name/py/pybibget/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, python3
+, fetchPypi
+}:
+
+let
+  pname = "pybibget";
+  version = "0.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1n4as3kdsgrcmxxyjz9klm0v0pb24rsa1b7r43j7wmnfsir8i81k";
+  };
+
+in
+python3.pkgs.buildPythonApplication  {
+  inherit pname version src;
+  pyproject = true;
+
+  propagatedBuildInputs = with python3.pkgs; [
+    lxml
+    httpx
+    appdirs
+    aiolimiter
+    pybtex
+    pylatexenc
+    numpy
+    networkx
+    requests
+  ];
+
+  # Assuming you have tests and they can be run by pytest
+  # If not, you might want to set `doCheck = false;`
+  # or adjust the checkPhase to properly run the package's tests.
+  doCheck = false;
+  #checkInputs = [ pytestCheckHook ];
+  #pythonImportsCheck = [ "pybibget" ];
+
+  meta = with lib; {
+    description = "Command line utility to automatically retrieve BibTeX citations from MathSciNet, arXiv, PubMed and doi.org";
+    homepage = "https://github.com/wirhabenzeit/pybibget";
+    license = licenses.mit;
+    maintainers = with maintainers; [ vasissualiyp ];
```

```suggestion
  meta = {
    description = "Command line utility to automatically retrieve BibTeX citations from MathSciNet, arXiv, PubMed and doi.org";
    homepage = "https://github.com/wirhabenzeit/pybibget";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ vasissualiyp ];
```

Tracking issue: #292468
Any uses of `lib` are better "pushed down"